packages/react-menu/src/components/MenuItem/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './MenuItem';
-export * from './MenuItem.types';
+// Explicit exports to ommit MenuItemCommon
+export type { MenuItemSlots, MenuItemProps, MenuItemState } from './MenuItem.types';
```

Comment:
nit: Order these alphabetically
```suggestion
export type { MenuItemProps, MenuItemSlots, MenuItemState } from './MenuItem.types';
```